### PR TITLE
controller-deployment: add image secret option to yaml, add services …

### PR DESCRIPTION
**Pull request by michalrabinowitch**
…configuration values

-Allow pulling controller image from user defined repo (current pull is from public repo)
-Allow controller services to be configured: service type and service naming convention

### CODE REVIEW

**Comment by ushachar** [comment]
Ticket # please...

File: redis-enterprise/values.yaml [hunk 1]
```diff
@@ -11,6 +11,10 @@ redisControllerImage:
   tag: latest
   pullPolicy: IfNotPresent
 
+redisControllerConfiguration:
```
Comment (danni-m) [inline redis-enterprise/values.yaml]: Add documentation to README

File: redis-enterprise/values.yaml [hunk 2]
```diff
@@ -44,7 +48,10 @@ persistentVolume:
 externalUIServiceAnnotations: {}
 
 openShiftDeployemnt: false
-#imagePullSecrets:
+#redisEnterpriseImagePullSecrets:
```
Comment (danni-m) [inline redis-enterprise/values.yaml]: no need for double pull secrets

**Review by danni-m — CHANGES_REQUESTED** [review]

**Comment by michalrabinowitch** [comment]
@danni-m pushed changes

**Review by danni-m — APPROVED** [review]

**Review by danni-m — APPROVED** [review]

**Pull request merged**